templates/shared/secrets/add_list.tmpl

```diff
@@ -0,0 +1,68 @@
+<h4 class="ui top attached header">
+	{{.locale.Tr "secrets.secrets"}}
+	<div class="ui right">
+		<div class="ui primary tiny show-panel button" data-panel="#add-secret-panel">{{.locale.Tr "secrets.creation"}}</div>
+	</div>
+</h4>
+<div class="ui attached segment">
+	<div class="{{if not .HasError}}hide {{end}}mb-4" id="add-secret-panel">
+		<form class="ui form" action="{{.Link}}" method="post">
+			{{.CsrfTokenHtml}}
+			<div class="field">
+				{{.locale.Tr "secrets.description"}}
+			</div>
+			<div class="field{{if .Err_Title}} error{{end}}">
+				<label for="secret-title">{{.locale.Tr "secrets.name"}}</label>
+				<input id="secret-title" name="title" value="{{.title}}" autofocus required pattern="^[a-zA-Z_][a-zA-Z0-9_]*$" placeholder="{{.locale.Tr "secrets.creation.name_placeholder"}}">
+			</div>
+			<div class="field{{if .Err_Content}} error{{end}}">
+				<label for="secret-content">{{.locale.Tr "secrets.value"}}</label>
+				<textarea id="secret-content" name="content" required placeholder="{{.locale.Tr "secrets.creation.value_placeholder"}}">{{.content}}</textarea>
+			</div>
+			<button class="ui green button">
+				{{.locale.Tr "secrets.creation"}}
+			</button>
+			<button class="ui hide-panel button" data-panel="#add-secret-panel">
+				{{.locale.Tr "cancel"}}
+			</button>
+		</form>
+	</div>
+	{{if .Secrets}}
+	<div class="ui key list">
+		{{range .Secrets}}
+		<div class="item">
+			<div class="right floated content">
+				<button class="ui red tiny button delete-button" data-url="{{$.Link}}/delete" data-id="{{.ID}}">
+					{{$.locale.Tr "settings.delete_key"}}
+				</button>
+			</div>
+			<div class="left floated content">
+				<i>{{svg "octicon-key" 32}}</i>
+			</div>
+			<div class="content">
+				<strong>{{.Name}}</strong>
+				<div class="print meta">******</div>
```

Comment:
Do we need this? It does not add a benefit for the user.

Reply:
How does it render?

Reply:
It looks like this:
![](https://user-images.githubusercontent.com/9418365/207823319-b8a4903f-38ca-4af7-9d05-336a5af906f3.png)

Reply:
Hmm, we should replace icon with `octicon-key-asterisk`, remove the `***` line and somehow slim down the view. The whole list wastes too much vertical space imho.

Reply:
Github:
![grafik](https://user-images.githubusercontent.com/1666336/209005101-8338f751-df46-4526-ba09-76d2ba2f6cdc.png)

GitLab:
![grafik](https://user-images.githubusercontent.com/1666336/209005718-6bb48d8f-398a-4f5f-8681-1c24ba94bda2.png)
GitLab has a button which reveals the value, replacing the ***** with the original string.

Reply:
I'd prefer a simple compact list like GitHub, but even more compact. `octicon-lock` is fine as well I suppose. Does your change affect all three secret lists?

<img width="841" alt="image" src="https://user-images.githubusercontent.com/115237/209006198-342f46bb-9aa3-4afb-b763-19956d2ef600.png">